test/functional/test_runner.py

```diff
@@ -83,6 +83,7 @@
     'feature_pruning.py',
     'feature_dbcrash.py',
     'feature_index_prune.py',
+    'wallet_pruning.py',
```

Comment:
In 6abd81b11dd1a45bdffcc85ba90591474a3f48b0 "test: Wallet imports on pruned nodes"

Should have `--legacy-wallet` to indicate that this test is a legacy wallet only test. Also, the default is `--descriptors`.

```suggestion
    'wallet_pruning.py --legacy-wallet',
```